.github/workflows/ci.yml

```diff
@@ -6,10 +6,12 @@ on:
 
 jobs:
   PHPUnit:
-    name: PHPUnit (PHP ${{ matrix.php }})
+    name: PHPUnit (PHP ${{ matrix.php }}; RDBMS ${{ matrix.rdbms }})
```

Comment:
Nit: Job title should be shortened:

![Screenshot 2025-05-21 at 18-19-35 Add rdbms matrix for wider compatibility testing by WyriHaximus · Pull Request #196 · friends-of-rea](https://github.com/user-attachments/assets/11cb2098-3395-43cf-a3fa-4fd38d85596c)



```suggestion
    name: PHPUnit (PHP ${{ matrix.php }} + ${{ matrix.rdbms }})
```

Reply:
Yup that looks a whole lot better: 
![image](https://github.com/user-attachments/assets/f3e66fdf-91f1-4855-946d-191e51036315)